app/utils.py

```
@@ -14,4 +16,56 @@ def get_gravatar_url(email, size=200, default='identicon'):
+def check_for_updates():
+    """Check if the current version is outdated."""
+    current_version = os.getenv("APP_VERSION", "unknown")
+    latest_version = get_latest_release()
+
+    if current_version == "unknown" or latest_version == "unknown":
+        return None # Unable to check
+
+    # Parse versions to compare them semantically
+    def parse_version(version_str):
+        # Extract version numbers from strings like "v1.2.3" or "1.2.3"
+        match = re.search(r'v>(\d+)(?:\.(\d+))?(?:\.(\d+))?', version_str)
```

Ruff (0.8.2)

Undefined name `re`

(F821)